src/app/layouts/nav-list/nav-list.component.html

```diff
@@ -0,0 +1,57 @@
+<td-layout-nav-list #navList [opened]="media.registerQuery('gt-sm') | async" [mode]="(media.registerQuery('gt-sm') | async) ? 'side' : 'over'"
+  [sidenavWidth]="(media.registerQuery('gt-xs') | async) ? '350px' : '100%'">
+  <div td-sidenav-toolbar-content layout="row" layout-align="start center">
+    <button mat-icon-button tdLayoutToggle>
+      <mat-icon>menu</mat-icon>
+    </button>
+    <mat-icon [routerLink]="['/']" class="mat-icon-logo cursor-pointer" svgIcon="assets:teradata"></mat-icon>
+    <span [routerLink]="['/']" class="cursor-pointer text-upper">Sandbox</span>
+  </div>
+  <div td-toolbar-content layout="row" layout-align="start center" flex>
+    <button mat-icon-button tdLayoutNavListOpen [hideWhenOpened]="true">
+      <mat-icon>arrow_back</mat-icon>
+    </button>
+    <span>Nav List</span>
+    <span flex></span>
+    <a mat-icon-button matTooltip="Docs" matTooltipPosition="before" matTooltipDelay="500" href="https://teradata.github.io/covalent/" target="_blank">
+      <mat-icon>chrome_reader_mode</mat-icon>
+    </a>
+    <a mat-icon-button matTooltip="Github" matTooltipPosition="before" matTooltipDelay="500" href="https://github.com/Teradata/covalent-resources" target="_blank">
+      <mat-icon svgIcon="assets:github"></mat-icon>
+    </a>
+    <a mat-icon-button [href]="baseURL + '/layouts/nav-list/nav-list.component.html'" target="_blank">
+      <mat-icon matTooltip="Open in Stackblitz" matTooltipPosition="before" matTooltipDelay="500">code</mat-icon>
+    </a>
+  </div>
+  <mat-nav-list td-sidenav-content [tdLayoutNavListClose]="!media.query('gt-sm')">
+    <ng-template let-item let-last="last" ngFor [ngForOf]="navmenu">
+      <a mat-list-item>
+        <mat-icon matListAvatar>{{item.icon}}</mat-icon>
+        <h3 matLine> {{item.title}} </h3>
+        <p matLine> {{item.description}} </p>
+      </a>
+      <mat-divider [inset]="true" *ngIf="!last"></mat-divider>
+    </ng-template>
+  </mat-nav-list>
+  <div tdMediaToggle="gt-xs" [mediaClasses]="['push-sm']">
+    <mat-card>
+      <div layout="row" layout-align="start center">
+        <div layout="column">
+          <mat-card-title flex>Main Content</mat-card-title>
+          <mat-card-subtitle flex>content area</mat-card-subtitle>
+        </div>
+        <span flex></span>
+      </div>
```

Comment:
No need for the extra 2 divs wrapping mat-card-title and subtitle. Same for the span.